ui.apps/src/main/content/jcr_root/apps/wknd/components/pagesearch/pagesearch.html

```diff
@@ -0,0 +1,21 @@
+<div data-sly-use.pagesearch="com.adobe.aem.guides.wknd.core.models.PageSearch"
+	 data-sly-use.placeholderTemplate="core/wcm/components/commons/v1/templates.html"
```

Comment:
this line does not do enything unless you refer somehow to it.
If you want u can take a look at image core component on how to use it
https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/image.html

The template is sometimes used for clear authoring experience - if component is not configured is rendered in proper (placeholder-like) way